lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_kuwait_when_residing_in_uk.govspeak.erb

```diff
@@ -30,13 +30,7 @@
     <%= render partial: 'partner_naturalisation_in_uk.govspeak.erb' %>
   <% end %>
 
-  <%= render partial: 'consular_fees_table.govspeak.erb',
-             locals: { calculator: calculator } %>
-
-  <%= render partial: 'link_to_consular_fees.govspeak.erb',
-             locals: { calculator: calculator } %>
-
-  <%= render partial: 'payment_information.govspeak.erb',
+  <%= render partial: 'os_affirmation/fees.govspeak.erb',
              locals: { calculator: calculator } %>
```

Comment:
Very minor: The commit note subject suggests it is changing one outcome (singular), but it is in fact changing two outcomes (plural).